README.md

```diff
@@ -46,27 +114,71 @@ Defining them is part of the set up of your project.
 - `DAPP_DID_URI` = _This is the URI of the Kilt DID that identifies your dApp_
 - `DAPP_NAME` = _This should be a custom name for your dApp_
 - `JWT_SIGNER_SECRET` = _This is secret key (string) that signs the Json-Web-Tokens before saving them in the Cookies_
 
-There is a script to facilitate the generation of the environment variables that are needed to set up your decentralized App (dApp).
-This script is called `./scripts/genesisEnvironmentVariables.ts`.
-You can execute it by running `yarn environment`.
+### How do I get mine?
+
+You have three different options:
 
-Setting up is a step by step process.
-You will need to run the script repeatedly and follow the instructions that it provides, depending on your project's state.
-After running this script each time, you need to manually copy the output and save it on the .env-file on the main project folder.
+1. Using `yarn environment`:
 
-## Process
+   There is a script to facilitate the generation of the environment variables inside of this repository.
+   This script is called `./scripts/genesisEnvironmentVariables.ts`.
+
+   - You can execute it by running `yarn environment`.
+
+   Setting up your environment with this script is a step by step process.
+   You will need to run the script repeatedly and follow the instructions that it provides, depending on your project's state.
+   After running this script each time, you need to manually copy the output and save it on the `.env`-file .
+
+2. Using the [kilt-distillery-cli](https://github.com/KILTprotocol/kilt-distillery-cli)
+
+   This is a _Command Line Interface_ tool that can help you obtaining this variables and also does other common tasks (unrelated to this project).
+   The distillery uses the same key derivation as this repository, which means that it is highly compatible.
+
+3. Without help:
+
+   If you are a _pro_, you could defined and generate them externally and add them to the `.env`-file.
+   You would probably have to modify the `generateKeyPairs.ts` files (on _scripts_ and _backend_) to match your key derivation though.
+
+## Well-Known-DID-Configuration
+
+After having all your environment variables, to build a **Domain Linkage Credential**:
+
+- run: `yarn well-known-config`.
+
+If you want know more about this, check out the [Identity Foundation Documentation](https://identity.foundation/.well-known/resources/did-configuration/).
+[Our dApp documentation also have a section about this.](https://docs.kilt.io/docs/develop/dApp/well-known-did-config)
+
+## Starting the dApp locally
+
+After having all your environment variables and your well-known-did-configuration:
+
+- run `yarn start`.
+
+## Login Process
+
+After having set up the whole project, when the website is up and running, the login process can take place.
+Each user that wants to login would trigger the process that is displayed below.
+The process is full of HTTPS queries and extension-api's messages.
```

Comment:
If you run it locally it's not HTTPS. What should this sentence tell the developer? It currently conveys that the process contains many HTTPS queries with the connotation that it's very complicated.

I would rather avoid that.

Reply:
Rephrase it to:
`The process is full of interactions between the client (browser), the extension/wallet and the server (backend).`

Reply:
The problem is `full of`. This sounds like it's full. There is no space for another request or your computer would break.

Reply:
What do you want to tell the reader with that?

Reply:
The process consists of multiple extension-api messages that are transmitted via HTTP.

Try to use a neutral phrases. full of, abundant are all rather emotional terms. We don't want to convey emotions. Especially not the ones that don't contribute to a nice experience for the reader.

Reply:
The other critic was referring to the "S" on HTTP**S**, right?

Reply:
`abundant` and `full` are positive terms for me. 

But I can change it to multiple, if it makes you happier.